Sources/PointFree/Subscribe.swift

```diff
@@ -50,7 +49,7 @@ private func subscribe(_ conn: Conn<StatusLineOpen, Tuple2<SubscribeData, Databa
       .withExcept(const(unit))
       .flatMap { subscribeData in
         Current.stripe
-          .createCustomer(user, subscribeData.token, subscribeData.vatNumber.isEmpty ? nil : subscribeData.vatNumber)
+          .createCustomer(subscribeData.token, user.id.rawValue.uuidString, user.email, subscribeData.vatNumber)
```

Comment:
We used to pass `Database.User` to `createCustomer` but I've decoupled the two so that `Stripe` and `Database` don't have a circular dependency.